lib/pbench/server/database/models/users.py

```diff
@@ -72,13 +66,9 @@ def add(self):
             Database.db_session.rollback()
             raise e
 
-    # Prevent update on "registered_on" field
-    @validates("registered_on")
-    def evaluate_registered_on(self, key, value):
-        if self.registered_on:  # Field already exists
-            if self.registered_on.strftime("%Y-%m-%d %H:%M:%S.%f") != value:
-                raise ValueError("registered_on cannot be modified.")
-        return value
+    @validates("password")
+    def evaluate_password(self, key, value):
+        return generate_password_hash(value)
```

Comment:
This is actually interesting since you're using it as a filter rather than a "validator" ... this could also be used for `registered_on`, if `default=` doesn't work, by simply ignoring the input value and returning the timestamp. (I suppose it could also _fail_ if there's already a value that's not `None` and doesn't match the specified new `value`.)